otsclient/cmds.py

```diff
@@ -164,9 +177,7 @@ def is_timestamp_complete(stamp, args):
 
 def upgrade_timestamp(timestamp, args):
     """Attempt to upgrade an incomplete timestamp to make it verifiable
-
```

Comment:
Don't delete these two lines of whitespace; makes it hard to read the docstring.